docs/fields.md

```diff
@@ -2028,7 +2028,7 @@ Artifact indicates an artifact to place at a specified path
 |`fromExpression`|`string`|FromExpression, if defined, is evaluated to specify the value for the artifact|
 |`gcs`|[`GCSArtifact`](#gcsartifact)|GCS contains GCS artifact location details|
 |`git`|[`GitArtifact`](#gitartifact)|Git contains git artifact location details|
-|`globalName`|`string`|GlobalName exports an output artifact to the global scope, making it available as '{{io.argoproj.workflow.v1alpha1.outputs.artifacts.XXXX}} and in workflow.status.outputs.artifacts|
```

Comment:
The `status` portion is correct here, it does literally exist there.

The `io...outputs` is the part that needs to be modified. See https://github.com/argoproj/argo-workflows/pull/13702#discussion_r1790619752 for how to make that work with the codegen